Added Azure.Defender.Storage.SensitiveData (#2233)
Azure/PSRule.Rules.Azure: docs/en/rules/Azure.Defender.Storage.SensitiveData.md

---
severity: Critical
pillar: Security
category: Data protection
resource: Microsoft Defender for Cloud
online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Defender.Storage.SensitiveData/
---

# Sensitive data threat detection

## SYNOPSIS

Enable sensitive data threat detection in Microsoft Defender for Storage.

## DESCRIPTION

> Microsoft Defender for Storage provides additional security for storage accounts.
> 
> One of the features in the Defender for Storage service is sensitive data threat detection that is powered by the Sensitive Data Discovery engine that uses a smart sampling method to find storage accounts with sensitive data.
> 
> The service is integrated with Microsoft Purview's sensitive information types (SITs) and classification labels.
> 
> The sensitive data threat detection capability helps teams identify and prioritize data security incidents for faster response times. Defender for Storage alerts include findings of sensitivity scanning and indications of operations that have been performed on resources containing sensitive data.
> 
> This can be useful when:
> 
> - Reduce the likelihood of data breaches
> - Detect exposure events
> - Detect suspicious activities on resources containing sensitive data 
> 
> When enabling sensitive data threat detection, the sensitive data categories include built-in sensitive information types (SITs) in the default list of Microsoft Purview. This will affect the alerts received from Defender for Storage, storage or containers that are found with these SITs are marked as containing sensitive data.
> 
> It is also possible to customize the Data Sensitivity Discovery for a organization, you can create custom sensitive information types (SITs) and connect to your organizational settings.
> 
> Sensitive data threat detection in Microsoft Defender for Storage can be enabled at the subscription level and by doing so ensures all storage accounts in the subscription will be protected, including future ones.

Reviewer: When don't need to include everything from the docs, just enough to understand what the feature is and why they should enable it.

```suggestion
Sensitive data threat detection is an additional security feature for Microsoft Defender for Storage.
When enabled Defender for Storage provides alerts when sensitive data is discovered.

The sensitive data threat detection capability helps teams:

- Identity where sensitive data is stored.
- Detect possible security incidents resulting is data exposure.

When enabling sensitive data threat detection, the sensitive data categories include built-in sensitive information types (SITs) in the default list of Microsoft Purview. 
It is possible to customize the Data Sensitivity Discovery for a organization, by creating custom sensitive information types (SITs).

Sensitive data threat detection in Microsoft Defender for Storage can be enabled at the subscription level and by doing so ensures all storage accounts in the subscription will be protected, including future ones.
```
